platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -939,12 +942,57 @@ public void onRegisterFlingListener(MapboxMap.OnFlingListener listener) {
     }
   }
 
+  private class MapZoomControllerListener implements ZoomButtonsController.OnZoomListener {
+
+    private final MapGestureDetector mapGestureDetector;
+    private final UiSettings uiSettings;
+    private final Transform transform;
+
+    MapZoomControllerListener(MapGestureDetector detector, UiSettings uiSettings, Transform transform) {
+      this.mapGestureDetector = detector;
+      this.uiSettings = uiSettings;
+      this.transform = transform;
+    }
+
+    // Not used
+    @Override
+    public void onVisibilityChanged(boolean visible) {
```

Comment:
If it isn't used, should we apply the [interface-segregation principle (ISP)](https://en.wikipedia.org/wiki/Interface_segregation_principle)?
This is only a comment (no blocker at all), but something to take into account and fix in the future.